app/angular/service/authService.js

```diff
@@ -29,6 +29,7 @@ const authService = function ($http, $cookies) {
 
 	service.register = function (credentials) {
 		const body = {
+			"username": credentials.username,
```

Comment:
I've removed the username on purpose to not have to deal with [PII data](https://www.investopedia.com/terms/p/personally-identifiable-information-pii.asp), handle cookies and etc. For now I think it is better. I would appreciate if you remove the field from the screen instead.